### ISSUE TITLE
Code analysis and visual code representation

### ISSUE THREAD

**Issue by mstrag** [WrocTypeScript/talks#69]
- **Your full name**: Mateusz Strąg
- **Target audience** (beginner/intermediate/advanced/everyone): Everyone
- **Estimated duration**: 30 min
- **Keywords**: Static analysis, dynamic analysis, analysing tools, code visualisation, IDE extensions

# Code analysis and visual code representation

### One-sentence summary
How to browse code graphically and clearly visualize semantic dependencies in TypeScript project?

### What's the format — is it a case study, a live coding session, a workshop or something else?
At the beginning I will tell about analysing code and available tools (with examples) second part will be focused on available extension for visualising typescript projects with short live demo session. 
### Tell us more about the talk
Current IDEs are dynamically evolving, however, we are all forced to browse code by looking at flat-file structures No information about the semantic dependencies between different units. I will show how to browse code graphically and clearly visualize semantic dependencies in TypeScript project which would result in speeding up the process of learning and understanding the code
### A few words about yourself
Software engineer at @VirtusLab. Mainly focused on data visualization on a fronted side. Involved in whole VirtusLab R&D department which results in publishing first extensions on a marketplace.

### How can we find you on social media?
https://twitter.com/MateuszStrag

### Would you be willing to have a Q/A session after the talk?
Highly recommended! 

### Do you mind if we record the event?
I'm 🆗 with it! 
### Is there anything we can help you with?
i'm good

**Comment by karol-majewski** [comment]
Sounds amazing @mstrag! Thank you for the proposal.

When would you be willing to give the talk? March/April?

**Comment by mstrag** [comment]
@karol-majewski for me April sounds perfect! :) but could You schedule it together with #68 ?

**Comment by hasparus** [comment]
Hey @karol-majewski, this is the part of April visit from Kraków, we've been talking about.

Spoiler: The demo in @mstrag's talk is super impressive..

**Comment by hasparus** [comment]
Hey @mstrag 👋 The April meetup is going to happen online on our YouTube stream. Are you okay with giving your talk remotely?

**Comment by mstrag** [comment]
Yes I'm okay with streaming and recording as well, so see You on YT 💪
